GliaWidgets/Sources/View/Chat/ChatView.swift

```diff
@@ -162,7 +162,17 @@ class ChatView: EngagementView {
         ])
 
         addSubview(messageEntryView)
-        messageEntryViewBottomConstraint = messageEntryView.layoutIn(safeAreaLayoutGuide, edges: .bottom).first
+        let messageEntryInsets = UIEdgeInsets(
+            top: 0,
+            left: 0,
+            bottom: 4.5,
```

Comment:
Will not it cause some tiny diff on further runs? Could we round this value to be sure that there will not be this disturbing tiny diff? Wdyt?

Reply:
Let's see how CI will handle it. I will make adjustment if this will cause issues.